.devcontainer/devcontainer.json

```diff
@@ -5,6 +5,7 @@
     },
     "remoteUser": "root",
     "postCreateCommand": "yarn install",
+    "postStartCommand": "yarn start",
```

Comment:
This is not ideal because the user doesn't have access to the shell running `yarn start` which means they can't stop it. It's better that we tell the user they run `yarn start` themselves.

Reply:
Resolved in [011deaa](https://github.com/daytonaio/docs/pull/15/commits/011deaad823606a6a1bd5df423753b7084ec1314)